### PR TITLE
Add initial projects

### PR DESCRIPTION
**Pull request by undiscoveredimposter**
Add:
EasyCosmos
EasyCosmos.Tests
solution file

### CODE REVIEW

File: EasyCosmos.Tests/MigrationExtensionsTests.cs [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            foreach (var migrationType in migrations)
+            {
+                Assert.True(migrationType.IsSubclassOf(typeof(CosmosMigration)));
+            }
+        }
+
+        [Fact]
+        public void OrdersMigrationsByNumericalOrder()
+        {
+            var migrations = new [] {
```
Comment (creyke) [inline EasyCosmos.Tests/MigrationExtensionsTests.cs:25]: Probably worth flipping these so a sorting operation needs to take place for this test to pass.

File: EasyCosmos/ObjectActivator.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Linq.Expressions;
+
+namespace EasyCosmos
+{
+    class ObjectActivator
+    {
+        public static object CreateInstance(Type expectedType)
+        {
+            return Expression.Lambda<Func<object>>(Expression.New(expectedType)).Compile().Invoke();
```
Comment (creyke) [inline EasyCosmos/ObjectActivator.cs:10]: Any reason this over the built in Activator.CreateInstance() ?

**Review by creyke — CHANGES_REQUESTED** [review]

File: EasyCosmos/ObjectActivator.cs (hunk 2 above)
Comment (undiscoveredimposter) [reply EasyCosmos/ObjectActivator.cs:10]: performance gains, should be faster.
https://stackoverflow.com/questions/6582259/fast-creation-of-objects-instead-of-activator-createinstancetype

File: EasyCosmos.Tests/MigrationExtensionsTests.cs (hunk 1 above)
Comment (undiscoveredimposter) [reply EasyCosmos.Tests/MigrationExtensionsTests.cs:25]: The sorting doesn't happen when you call this method with a list of specific types or instances. This way you can create specific lists of migrations, such as a pre execution list and/or a post execution list. 
Though this does raise the problem that this test is a false positive and actually unnecessary  

Edit: will change to assert the run order by another way

**Review by creyke — APPROVED** [review]

**Pull request merged**